src/services/mentors.js

```diff
@@ -364,6 +364,8 @@ module.exports = class MentorsHelper {
 				...data,
 				...saasPolicyData,
 				visible_to_organizations: userOrgDetails.data.result.related_orgs,
+				visibility: organisationPolicy.mentee_visibility_policy,
+				external_mentee_visibility: organisationPolicy.external_mentee_visibility_policy,
```

Comment:
@Prajwal17Tunerlabs  this function creates mentor extension then why external_mentee_visibility is required here ?